angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/nimble-table-column-anchor.directive.ts

```diff
@@ -0,0 +1,139 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import type { TableColumnAnchor } from '@ni/nimble-components/dist/esm/table-column/anchor';
+import { BooleanValueOrAttribute, NumberValueOrAttribute, toBooleanProperty, toNullableNumberProperty } from '../../utilities/template-value-helpers';
+
+export type { TableColumnAnchor };
+
+/**
+ * Directive to provide Angular integration for the table column element for links.
+ */
+@Directive({
+    selector: 'nimble-table-column-anchor'
+})
+export class NimbleTableColumnAnchorDirective {
```

Comment:
Can't extend `NimbleAnchorBaseDirective` because this doesn't have a `href` property.